website/docs/components/modal/partials/code/how-to-use.md

```diff
@@ -1,22 +1,20 @@
-#### Browser support
+## Browser support
 
 The `Hds::Modal` component leverages the `<dialog>` element which is currently supported by all major browser vendors. To ensure support on older browser versions (for example, Safari 14 or Internet Explorer 11) we rely on [a polyfill](https://github.com/GoogleChrome/dialog-polyfill) that is automatically loaded when needed.
 
-#### Page scroll
+## Page scroll
 
-When an `Hds::Modal` component is open the rest of the page becomes inert. However, scrolling at the page level is still available. To make it clear to the users that the underlying elements are not interactive and to avoid confusion, we recommend disabling the page scroll `onOpen` and enabling it back `onClose` (for example, by setting `overflow: hidden;` and `overflow: auto;` respectively) by applying it to the `<body>` element.
+When an `Hds::Modal` component is open the rest of the page is disabled. However, scrolling at the page level is still available. To make it clear to the user that the underlying elements are not interactive and to avoid confusion, we recommend disabling the page scroll `onOpen` and enabling it back `onClose` (for example, by setting `overflow: hidden;` and `overflow: auto;` respectively) by applying it to the `<body>` element.
```

Comment:
```suggestion
When an `Hds::Modal` component is open, the rest of the page is disabled(via `inert`). However, scrolling at the page level is still available. To make it clear to the user that the underlying elements are not interactive and to avoid confusion, we recommend disabling the page scroll `onOpen` and enabling it back `onClose` (for example, by setting `overflow: hidden;` and `overflow: auto;` respectively) by applying it to the `<body>` element.
```